src/app/pages/wallets/my-wallets/pages-wallets-my-wallets.component.spec.ts

```diff
@@ -1,36 +1,52 @@
-import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { ComponentFixture, fakeAsync, flushMicrotasks, TestBed } from '@angular/core/testing';
 import { PagesWalletsMyWalletsComponent } from './pages-wallets-my-wallets.component';
 import { MaterialModule } from '../../../material.module';
 import { PagesWalletsMyWalletsService } from './pages-wallets-my-wallets.service';
 import SpyObj = jasmine.SpyObj;
+import createSpy = jasmine.createSpy;
 import createSpyObj = jasmine.createSpyObj;
-import { Subject } from 'rxjs';
+import { of, Subject } from 'rxjs';
 import { MyWallet } from './pages-wallets-my-wallet.model';
 import { IWalletApiResponse } from '../../../domains/wallets/domains.wallets.types';
-import { WALLET_RESP_MOCK } from 'src/app/domains/wallets/domains.wallets.mocks';
+import { WALLET_INSTANCE_MOCK, WALLET_RESP_MOCK } from 'src/app/domains/wallets/domains.wallets.mocks';
+import { SystemNotificationsService } from 'src/app/common/utils/system-notifications/system-notifications.service';
+import { By } from '@angular/platform-browser';
+import { MatDialog, MatDialogRef } from '@angular/material/dialog';
 
 describe('PagesWalletsMyWalletsComponent', () => {
   let component: PagesWalletsMyWalletsComponent;
   let fixture: ComponentFixture<PagesWalletsMyWalletsComponent>;
   let myWalletsServiceMock: SpyObj<PagesWalletsMyWalletsService>;
+  let systemNotificationsServiceMock: SpyObj<SystemNotificationsService>;
+  let matDialogMock: SpyObj<MatDialog>;
   let walletResp: IWalletApiResponse;
   let walletsSubject: Subject<MyWallet[]>;
   let walletSubject: Subject<MyWallet>;
+  let walletInstance: MyWallet;
+  let dialogAfterClosedSpy: jasmine.Spy;
 
   beforeEach(async () => {
     walletsSubject = new Subject<MyWallet[]>();
     walletSubject = new Subject<MyWallet>();
     walletResp = WALLET_RESP_MOCK;
+    walletInstance = WALLET_INSTANCE_MOCK;
+
     myWalletsServiceMock = createSpyObj<PagesWalletsMyWalletsService>(PagesWalletsMyWalletsService.name, ['getMyWallets', 'createWallet', 'updateWallet']);
     myWalletsServiceMock.getMyWallets.and.returnValue(walletsSubject.asObservable());
     myWalletsServiceMock.createWallet.and.returnValue(walletSubject.asObservable());
     myWalletsServiceMock.updateWallet.and.returnValue(walletSubject.asObservable());
 
+    dialogAfterClosedSpy = createSpy('afterClosed');
+    matDialogMock = createSpyObj<MatDialog>(MatDialog.name, ['open']);
```

Comment:
So this file is causing some eslint issues to fix those please do two things:
- enable this commented out linter in config file, we will keep it
- you need to remove `any` from line 41, to do this you need to remove `dialogAfterClosedSpy` and replace it with:
```typescript
matDialogRef = createSpyObj<MatDialogRef<WalletFormModalComponent>>(MatDialogRef.name, ['afterClosed']);
matDialogMock.open.and.returnValue(matDialogRef);
```
in tests you can use it like this:
```typescript
// instead this
dialogAfterClosedSpy.and.returnValue(of(undefined));

// do this
matDialogRef.afterClosed.and.returnValue(of(undefined));
```